src/CFamily/Rules/Resources.Designer.cs

```diff
@@ -69,6 +69,15 @@ internal class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Note: the following CFamily rules are not available in SonarLint: {0}.
+        /// </summary>
+        internal static string CFamily_RulesUnavailableInSonarLint {
```

Comment:
This string was previously (incorrectly) in `Core`. I should have moved it earlier when moving the code, but missed it.

I saw a few other strings that are still in `Core` for other classes that should not be. I'll move them separately.